addons/test_mail/tests/test_performance.py

```diff
@@ -832,6 +832,38 @@ def test_message_format(self):
             for message in res:
                 self.assertEqual(len(message['attachment_ids']), 2)
 
+    @mute_logger('odoo.tests', 'odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    @users('emp')
+    @warmup
+    def test_message_format_query(self):
+        name_field = self.env['ir.model.fields']._get(self.container._name, 'name')
```

Comment:
Why this line ? Does not seem to be used

Reply:
Copy/past mistake :-1: